tests/e2e/etcd_config_test.go

```diff
@@ -411,10 +411,7 @@ func TestEtcdHealthyWithTinySnapshotCatchupEntries(t *testing.T) {
 	for i := 0; i < 10; i++ {
 		clientId := i
 		g.Go(func() error {
-			cc, err := e2e.NewEtcdctl(epc.Cfg.Client, epc.EndpointsGRPC())
```

Comment:
Grpc e2e failing with data race after this removal. I'm not sure we can remove this instance of the old approach as the test is intended to simulate 10 different clients?

Reply:
`Etcdctl` is not a client, it a wrapper over a etcdctl binary. Not only each `Etcdctl` call is unique, but also method call is unique execution of binary

Reply:
Ahh right I see the spawns in `etcdctl.go`, apologies.  Any ideas why this change would be creating the data race with that in mind?

Reply:
Simple, I removed  scoped `err` variable.

Reply:
Nice and checks now green. Thanks for indulging the questions.